packages/flex-plugin-e2e-tests/src/core/parameters.ts

```diff
@@ -47,23 +55,34 @@ const { TWILIO_REGION } = process.env;
 const pluginName = 'flex-e2e-tester-plugin';
 const consoleBaseUrl = TWILIO_REGION ? `https://www.${TWILIO_REGION}.twilio.com` : 'https://www.twilio.com';
 const hostedFlexBaseUrl = TWILIO_REGION ? `https://flex.${TWILIO_REGION}.twilio.com` : 'https://flex.twilio.com';
+const operatingSystem = platform();
+
+const accountSid = `TWILIO_ACCOUNT_SID_${operatingSystem}`;
+const authToken = `TWILIO_AUTH_TOKEN_${operatingSystem}`;
 
 // These are required parameters - verify otherwise throw an error
-const requiredEnvs = ['TWILIO_ACCOUNT_SID', 'TWILIO_AUTH_TOKEN', 'CONSOLE_EMAIL', 'CONSOLE_PASSWORD'];
+const requiredEnvs = [accountSid, authToken, 'CONSOLE_EMAIL', 'CONSOLE_PASSWORD'];
```

Comment:
Don't we want to do the same thing with the email and password? Shouldn't they be different for each platform?

Reply:
We are good here. Since we are passing account sid to service-login we are specifying which project we are accessing. If you use the same Console credentials in two different browsers/environments, you can log into different projects and refreshing doesn't break that.